laikaIO/src/main/scala/pink/cozydev/protosearch/analysis/IndexFormat.scala

```diff
@@ -50,10 +51,15 @@ case object IndexFormat extends TwoPhaseRenderFormat[Formatter, BinaryPostProces
             output: BinaryOutput[F],
             config: OperationConfig,
         ): F[Unit] = {
-          val strs: List[String] = result.allDocuments.map(_.content).toList
+          val allDocs = result.allDocuments.toList
 
           val analyzer = Analyzer.default.withLowerCasing
-          val index = MultiIndex(Map("body" -> Index(strs.map(analyzer.tokenize))), "body")
+          val index = MultiIndex[RenderedDocument](
+            "body",
+            (Field("body", analyzer, true, true), _.content),
+            (Field("title", analyzer, true, true), d => renderTitle(d.title, d.path)),
+            (Field("path", analyzer, true, true), _.path.name),
```

Comment:
This path isn't a directly linkable thing yet. I should perhaps learn more about the Laika NavigationItem class....